handlers/species.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const Formatters = require('../lib/formatters');
+
+function errorHandler(reply, e) {
+  return this.helpers.errorHandler.apply(this, arguments);
+}
+
+// GET /species
+exports.listSpecies = async function(request, reply) {
+  try {
+    const species = await this.models.Specie.listSpecies();
```

Comment:
(Aqui se comenta directamente el codigo?)
me refiero a aquí, el this aquí que significa? ese this da acceso a todo?

Reply:
This es el scope de la funcion, en este caso el handler. Hay ciertas cosas que le inyectamos al this con hapi, por ejemplo los modelos y los helpers. Hay cosas que hapi inyecta al this de igual forma. Pero en los controladores (handlers), lo usamos principalmente para acceder a los modelos y los helpers